pkgs/by-name/cr/crossplane-cli/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "crossplane-cli";
+  version = "1.14.0";
+
+  src = fetchFromGitHub {
+    owner = "crossplane";
+    repo = "crossplane";
+    rev = "refs/tags/v${version}";
```

Comment:
Please remove `refs/tags/`

Reply:
why?

Reply:
I have never seen it before. It doesn't seem necessary.

Reply:
can we not do pointless nits? a quick search in this repo yields 2800+ results.

Reply:
Not a problem, fixed :)